addons/mrp/models/mrp_workorder.py

```diff
@@ -767,7 +767,8 @@ def _get_conflicted_workorder_ids(self):
                 AND wo2.state IN ('pending','ready')
                 AND wo1.id != wo2.id
                 AND wo1.workcenter_id = wo2.workcenter_id
-                AND (wo2.date_planned_start, wo2.date_planned_finished) OVERLAPS (wo1.date_planned_start, wo1.date_planned_finished)
+                AND (DATE_TRUNC('second', wo2.date_planned_start), DATE_TRUNC('second', wo2.date_planned_finished))
```

Comment:
if it effecively round instead of trunc in the web client you could make something like this : https://stackoverflow.com/questions/6195439/postgres-how-do-you-round-a-timestamp-up-or-down-to-the-nearest-minute to have the same behavior

Reply:
It is trunc, I updated the commit message! Thanks